src/helpers/is-user-core-member.ts

```diff
@@ -18,9 +18,9 @@ import { getCoreMemberLogins } from '../utils/get-core-member-logins';
 
 export class IsUserCoreMember extends HelperInputs {}
 
-export const isUserCoreMember = async ({ pull_number, login = context.actor }: IsUserCoreMember) => {
+export const isUserCoreMember = async ({ pull_number, login = context.actor, codeowners_overrides }: IsUserCoreMember) => {
```

Comment:
basically just bringing in the param, already defined in `HelperInputs`, to this helper